src/frontend/src/lib/utils/tokens.utils.ts

```diff
@@ -80,3 +80,11 @@ export const pinTokensWithBalanceAtTop = ($tokens: TokenUi[]): TokenUi[] => {
 		...nonPositiveBalances
 	];
 };
+
+/**
+ * Calculates total USD balance of the provided tokens list.
+ *
+ * @param $tokens - The list of tokens for total USD balance calculation.
+ */
+export const getTokensTotalUsdBalance = ($tokens: TokenUi[]): number =>
```

Comment:
I think @AntonioVentilii-DFINITY uses the `$` pattern to express the fact that the variable is a store. In this case it isn't one so I would remove this prefix but, Antonio can confirm?

Reply:
True, this helper can possible called for any list of tokens, not necessarily a store value from `network-tokens.derived`. @AntonioVentilii-DFINITY fine with you if I remove `$` here?

Reply:
If the scope is not purely for stores or drives, I agree, you can remove the dollar sign

Reply:
Renamed 👍